runtime/vm/exceptiondescribe.c

```diff
@@ -165,46 +162,35 @@ printStackTraceEntry(J9VMThread * vmThread, void * voidUserData, UDATA bytecodeO
 			sourceFileNameLen = LITERAL_STRLEN("Unknown Source");
 		}
 		if (NULL != moduleNameUTF) {
-			if (NULL != moduleVersionUTF) {
-				if (lineNumber) {
-					format = j9nls_lookup_message(
-						J9NLS_DO_NOT_PRINT_MESSAGE_TAG,
-						J9NLS_VM_STACK_TRACE_WITH_MODULE_VERSION_LINE,
-						"\tat %.*s.%.*s (%s@%s/%.*s:%u)\n");
-				} else {
-					format = j9nls_lookup_message(
-						J9NLS_DO_NOT_PRINT_MESSAGE_TAG,
-						J9NLS_VM_STACK_TRACE_WITH_MODULE_VERSION,
-						"\tat %.*s.%.*s (%s@%s/%.*s)\n");
-				}
+			if (lineNumber) {
+				format = j9nls_lookup_message(
+					J9NLS_DO_NOT_PRINT_MESSAGE_TAG,
+					J9NLS_VM_STACK_TRACE_WITH_MODULE_VERSION_LINE,
+					(NULL == moduleVersionUTF) ? "\tat %.*s.%.*s (%s@%u/%.*s:%u)\n" : "\tat %.*s.%.*s (%s@%s/%.*s:%u)\n");
```

Comment:
This can be avoided/simplified by using a macro, see This can be done via a macro, see https://github.com/eclipse-openj9/openj9/blob/master/runtime/vm/vmprops.c#L578-L580

Reply:
Yeah, use `J9_STR(JAVA_SPEC_VERSION)`.